client/src/components/_common/Sidebar/Sidebar.module.css

```diff
@@ -10,8 +10,8 @@
   padding-top: 20px;
 }
 
-/* when there is a sidebar inside a section */
-:global(.c-section--has-content-layout-has-sidebar) main {
+/* when there is a sidebar directly inside a section */
+:global(.c-section--has-content-layout-has-sidebar) > main {
```

Comment:
This change prevents a nested `<main>` tag from having its left-side padding removed.

<sup>P.S. Having a second `<main>` is a neglected bug. There is only supposed to be one `<main>` tag per document.</sup>